assets/javascripts/modules/SSO_links.js

```diff
@@ -36,7 +36,14 @@ module.exports = function(element, ssoUrl, ssoMethod) {
 
     if (clientSso || serverSso) {
 
-      elementHref = element.href;
+      destinationUrl = element.formaction || element.href;
+
+      //if destinationUrl is still not defined then the elemnt should be a button and we are using a non HTML5 compliant browser that cannot access the element.formaction attribute
+      if (!destinationUrl) {
+        var buttonParentForm = $element.closest('form')[0];
+        destinationUrl = (buttonParentForm) ? buttonParentForm.action : '';
+      }
+
```

Comment:
lets use `element.form.action` as this is cross browser compliant all you need here is:

```
destinationUrl = (element.form && element.form.action) || element.href;
```